docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -49,105 +49,1308 @@ SETTINGS cast_keep_nullable = 1
 └──────────────────┴─────────────────────┴──────────────────┘
 ```
 
-## toInt(8\|16\|32\|64\|128\|256)
+## toInt8
 
-Converts an input value to a value the [Int](../data-types/int-uint.md) data type. This function family includes:
+Converts an input value to a value of type `Int8`.
```

Comment:
To distinguish this function from its `Or*` variants, we could add a sentence: `Throws an exception in case of error.`.